apps/modernization-ui/src/apps/page-builder/components/CreateQuestion/CreateQuestion.tsx

```diff
@@ -66,6 +67,7 @@ export type QuestionFormType = {
     HL7Segment?: string;
     relatedUnits?: string;
     allowFutureDates?: string;
+    codeSet: any;
```

Comment:
Issue: use of `any` is prohibited. (other than auto generated code cause its dumb)